### PR TITLE
Issue 5018: Node previews, wrong layout (add tests)

### PR DESCRIPTION
**Pull request by bugfolder**
Addresses [Issue 5018](https://github.com/backdrop/backdrop-issues/issues/5018).

Add tests for the proper layout to use on node preview pages.

Current Backdrop core will fail this test for node add page preview (but will pass for node edit preview).

[PR 3574](https://github.com/backdrop/backdrop/pull/3574) will pass it.

### CODE REVIEW

File: core/modules/layout/tests/layout.test [hunk 1]
```diff
@@ -2287,6 +2287,88 @@ class LayoutSelectionTest extends BackdropWebTestCase {
...
+    $body = $this->randomName();
+    $edit = array(
+      'title' => $title,
+      'body[und][0][value]' => $body,
+    );
+    $this->backdropPost(NULL, $edit, t('Save'));
+
+    // Edit the node with a new random body content, then preview.
+    $this->backdropGet('node/1/edit');
+   $body = $this->randomName();
```
Comment (klonos) [inline core/modules/layout/tests/layout.test]:
```suggestion
    $body = $this->randomName();
```

File: core/modules/layout/tests/layout.tests.info [hunk 2]
```diff
@@ -57,3 +63,8 @@ name = Layout Flexible Template Test
 description = Tests that flexible layout templates perform correctly.
 group = Layout
 file = layout.test
+
+; Added by Backdrop CMS packaging script on 2021-01-27
```
Comment (jenlampton) [inline core/modules/layout/tests/layout.tests.info:67]: We should make sure this info (added by the packaging script) doesn't accidentally end up committed to core.

File: core/modules/layout/tests/layout_test/layout_test.info [hunk 3]
```diff
@@ -5,3 +5,8 @@ version = BACKDROP_VERSION
 type = module
 backdrop = 1.x
 hidden = TRUE
+
+; Added by Backdrop CMS packaging script on 2021-01-27
```
Comment (jenlampton) [inline core/modules/layout/tests/layout_test/layout_test.info:9]: Same thing here, no version info needed in core (will be added by the packager later)

**Comment by bugfolder** [comment]
These tests were merged into https://github.com/backdrop/backdrop/pull/3574, so this PR is no longer needed.

**Comment by jenlampton** [comment]
Closing in favor of https://github.com/backdrop/backdrop/pull/3574

**Pull request closed without merging**